docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,199 @@
+IDs are valid.
+
+
+External networks overlap
+-------------------------
+
+Checks :ref:`external-networks-overlap`. Check starts only if
+``[migrate] migrate_ext_nets`` config option is set to ``True`` in the
+:ref:`primary-config-file`. This check verifies if there is any IP range
+overlapping between source and destination clouds' external networks. In
+particular, external subnets are compared. The main idea is to check if theirs
+allocation pools overlap. Only in this case migration will not start. If
+subnets have the same CIDRs, but different allocation pools, migration will
+proceed, because in this case there are no any IP conflicts. Example::
+
+    Network A (source cloud)
+    Subnet A
+    CIDR: 1.1.1.0/24
+    Allocation pool: 1.1.1.10 - 1.1.120
+
+    Network B (destination cloud)
+    Subnet B
+    CIDR: 1.1.1.0/25
+    Allocation pool: 1.1.1.5 - 1.1.1.25
+
+    Result:
+    Networks A and B overlap, cause they both have 1.1.1.10 - 1.1.1.25 IP range
+
+
+Networks segmentation IDs overlap
+---------------------------------
+
+Checks if network's segmentation ID is available on the destination. If it is
+available (not busy with any other network), segmentation ID will be kept.
+Otherwise new segmentation ID will be generated for migrated network.
+
+.. warning::
+
+    This check always passes. If segmentation ID is already used on the
+    destination cloud, ``Warning`` message will be shown during the check and
+    migration will proceed.
+
+
+Floating IPs overlap
+--------------------
+
+Checks if :ref:`Floating IPs <floating-ips>` from source cloud overlap with
+destination. If the same Floating IP address is found on the destination cloud,
+following attributes of Floating IPs will be compared:
+
+ - Tenant;
+ - External network;
+ - External network's tenant.
+
+Floating IP considered as overlapped, if any of above attributes differs,
+
+Also it checks if Floating IP is not busy (i.e. is not associated to VM on
+source and destination clouds at the same time) on both environments.
+
+
+Busy physical networks on DST of FLAT network type
+--------------------------------------------------
+
+A ``flat`` network is a network that does not provide any segmentation options.
+A traditional L2 ethernet network is a ``flat`` network. Any servers attached
+to this network are able to see the same broadcast traffic and can contact each
+other without requiring a router. ``flat`` networks are often used to attach
+Nova servers to an existing L2 network (this is called a ``provider network``).
+
+``physical_network`` parameter is required for creating ``flat`` network in
+Neutron. And this physical network can be used only once for any ``flat``
+network. So, if any ``flat`` network already exists on the cloud, there is no
+possibility to create another ``flat`` network with the same physical
+network. And this check exists for determining such sorts of busyness.
```

Comment:
I'd suggest that you split this into subsections:

 - Description of flat networks
 - Why checking those is important